sphinx/util/__init__.py

```diff
@@ -315,7 +315,7 @@ def get_module_source(modname):
     elif not (lfilename.endswith('.py') or lfilename.endswith('.pyw')):
         raise PycodeError('source is not a .py file: %r' % filename)
     elif '.egg' in filename:
-        eggpath, _ = re.split('(?<=\\.egg)/', filename)
+        eggpath, _ = re.split('(?<=\\.egg)/', filename, 1)
```

Comment:
This doesn't work (expanding a list of size 1 into 2 variables)!

I think you wanted to write `eggpath = re.split('(?<=\\.egg)/', filename, 1)[0]` (which I have tested and works fine)

Reply:
OMG. I also have to rewrite elif-clause like `elif '.egg/' in filename`...